php-cs-fixer

```diff
@@ -28,8 +28,8 @@ set_error_handler(static function ($severity, $message, $file, $line) {
         exit(1);
     }
 
-    if (\PHP_VERSION_ID < 70400 || \PHP_VERSION_ID >= 80200) {
-        fwrite(STDERR, "PHP needs to be a minimum version of PHP 7.4.0 and maximum version of PHP 8.1.*.\n");
+    if (\PHP_VERSION_ID < 70400 || \PHP_VERSION_ID >= 80300) {
```

Comment:
```suggestion
    if (\PHP_VERSION_ID < 7_04_00 || \PHP_VERSION_ID >= 8_03_00) {
```

Reply:
I explicitly left it with old syntax when I was migrating all of PHP version-strings, so if someone would run it on old PHP, they will get proper msg instead of "syntax error"

Reply:
oh, this notation is only PHP 8.1+

Reply:
is it? I see it is used in [psalm](https://github.com/vimeo/psalm/pull/9472/files), for example, which runs in lower versions, such as php 7.4

Reply:
[Underscore numeric separator](https://php.watch/versions/7.4/underscore_numeric_separator) was introduced in 7.4, so in fact we could use this notion 👍

Reply:
@Wirone I see. kubawerlos's point makes sense, though.